clinica/iotools/converters/aibl_to_bids/aibl_utils.py

```diff
@@ -692,7 +700,7 @@ def create_participants_df_AIBL(input_path, clinical_spec_path, clinical_data_di
         else:
             participant_df['sex'][i] = 'F'
 
-    participant_df.replace('-4', np.nan)
+    participant_df.replace('-4', "n/a")
```

Comment:
Though it was done in the past, I think that `-4` is more meaningful than `"n/a"`.

I'll let @nburgos comment on that but I would prefer letting `-4` since it corresponds to a test that was attempted, but for which no result could be obtained.

Reply:
In AIBL it is not the case, missing values are actually encoded as `-4` by the BIDS converter (instead of `nan`)

Reply:
My bad. In that case, you can set as resolved the comments regarding `-4`. I thought naively that `4` had the same meaning between ADNI and AIBL.

Reply:
It would have been great ! Unfortunately, it is not the case :(